commit 8a2f32c4521351096824e7c905bd8066ac4719e4 (refs/changes/15/3615/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-03-13T21:12:05+00:00 (1 year, 7 months ago)
    
    NET-387 portolan command line asks for --vnet_id but really wants --vnet-id

diff --git a/bin/portolan b/bin/portolan
index 4de1039..ae22bf1 100755
--- a/bin/portolan
+++ b/bin/portolan
@@ -70,7 +70,7 @@ var OPTS = {
     },
 
     vnet_id: {
-        names: [ 'vnet-id' ],
+        names: [ 'vnet_id' ],
         type: 'positiveInteger',
         help: 'Virtual network ID'
     },
